packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -18,17 +18,28 @@ const double _kCloseProgressThreshold = 0.5;
 const Color _kTransparent = const Color(0x00000000);
 const Color _kBarrierColor = Colors.black54;
 
-class _BottomSheetDragController extends StatelessComponent {
-  _BottomSheetDragController({
+class BottomSheet extends StatelessComponent {
+  BottomSheet({
     Key key,
-    this.performance,
-    this.child,
-    this.childHeight
-  }) : super(key: key);
+    this.performance, // will be mutated
```

Comment:
I meant that this object does this to the performance like call forward()
or reverse(). (As opposed to when you hand in a PerformanceView)

On Thu, Nov 19, 2015 at 8:24 PM Adam Barth <notifications@github.com> wrote:

> In packages/flutter/lib/src/material/bottom_sheet.dart
> <https://github.com/flutter/flutter/pull/508#discussion_r45434788>:
>
> >      Key key,
> > -    this.performance,
> > -    this.child,
> > -    this.childHeight
> > -  }) : super(key: key);
> > +    this.performance, // will be mutated
>
> Mutated? Maybe replaced?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flutter/flutter/pull/508/files#r45434788>.
>